swagger_to_sdk_config.json

```diff
@@ -1,26 +0,0 @@
-{
```

Comment:
@dw511214992 do we still need this for track2 SDK automation ?

Reply:
Brought these three files back